outputs.tf

```diff
@@ -17,3 +17,19 @@ output "client_configuration" {
   value       = data.awsutils_ec2_client_vpn_export_client_config.default.client_configuration
   description = "VPN Client Configuration data."
 }
+
+output "full_client_configuration" {
+  value = var.export_client_certificate ? templatefile(
+    "${path.module}/templates/client-config.ovpn.tpl",
```

Comment:
Let's parameterize `${path.module}/templates/client-config.ovpn.tpl` so that we use that if nothing else is passed.